admin_manual/enterprise_ios_app/publishing_ios_app_8.rst

```diff
@@ -2,4 +2,32 @@
 Testing Your New Branded iOS App
 ================================
 
-TODO
\ No newline at end of file
+You'll distribute the file with the :file:`.ipa` extension, like our example 
+:file:`Acme iOS App-3.4.201.ipa`, from your `<https://customer.owncloud.com>`_ 
+account to your beta testers. To do this you'll need a Mac computer, an iPhone 
+or iPad registered in your Apple developer account, and the iTunes account 
+associated with your Apple developer account.
+
```

Comment:
Not sure if it would be needed to add info about where the ipa will work, probably again (this is including on the provisioning creation process). I mean, if they are using an enterprise account, it will work on any device. However, if they are using a developer account, it will only work on those devices whose UDID are registered and included on the provisioning profiles.